app/controllers/api/v1/classifications_export_segments_controller.rb

```diff
@@ -0,0 +1,36 @@
+class Api::V1::ClassificationsExportSegmentsController < Api::ApiController
+  require_authentication :all, scopes: [:project]
+
+  before_action :require_access_to_project, only: [:create]
```

Comment:
Yeah, index and show are scoped through the `can_through_parent` stuff. But the `create` isn't using the normal stuff.